recipes/bzip2/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "1.0.6" %}
+
+package:
+  name: bzip2
+  version: {{ version }}
+
+source:
+  fn: bzip2-{{ version }}.tar.gz
+  url: http://www.bzip.org/{{ version }}/bzip2-{{ version }}.tar.gz
+  md5: 00b516f4704d4a7cb50a1d97e6e8e15b
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py>=35]
+
+requirements:
+  build:
+    - toolchain
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/bzlib.h  # [unix]
+    - test -f ${PREFIX}/lib/libbz2.a  # [unix]
```

Comment:
Also can we please add the CLI tests from PR ( https://github.com/conda-forge/staged-recipes/pull/1977 )?